### PR TITLE
Adding support for issues with labels

### PR DESCRIPTION
**Pull request by PhilLee-MSFT**
Need label support for 1ES dashboards.  Generalized a few properties/methods (and names) since the queries are not strictly milestone queries.

### CODE REVIEW

File: GitHubDashboard/Controllers/QueryController.cs [hunk 1]
```diff
@@ -74,21 +73,31 @@ public QueryController(GitHubClient gitHubClient)
...
             return issues;
         }
     }
 
+
     public class AssigneeCount
     {
-        public string assignee;
-        public int count;
+        public string Assignee;
```
Comment (Pilchie) [inline GitHubDashboard/Controllers/QueryController.cs]: I suspect that these changes will break the chart functionality at something like https://githubdashboard-dev-as.azurewebsites.net/assignedchart/dotnet/project-system?milestone=15.7

File: GitHubDashboard/Controllers/QueryController.cs [hunk 2]
```diff
@@ -47,15 +47,14 @@ public QueryController(GitHubClient gitHubClient)
...
+                Assignees = counts.Select(c => new AssigneeCount { Assignee = c.Key, Count = c.Value, }).OrderByDescending(a => a.Count).ToArray(),
             };
         }
 
-        private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone)
+        private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone, string labels)
         {
-            if (string.IsNullOrWhiteSpace(milestone) ||
-                milestone == "any")
+            if (string.IsNullOrWhiteSpace(milestone) || milestone == "any" || milestone == "undefined" || milestone == "*")
```
Comment (Pilchie) [inline GitHubDashboard/Controllers/QueryController.cs:57]: Where would you ever expect to get "undefined" from as a string?

**Review by Pilchie — CHANGES_REQUESTED** [review]

File: GitHubDashboard/Controllers/QueryController.cs (hunk 2 above)
Comment (PhilLee-MSFT) [reply GitHubDashboard/Controllers/QueryController.cs:57]: When the milestone is not set in the query params, the value will be 'undefined'.  This happens when only the milestone values are set.

File: GitHubDashboard/Controllers/QueryController.cs (hunk 1 above)
Comment (PhilLee-MSFT) [reply GitHubDashboard/Controllers/QueryController.cs]: OK.  I'll roll it back.  I didn't like the warnings.

**Review by Pilchie — APPROVED** [review]
Thanks for the contribution @PhilLee-MSFT !

**Pull request merged**